### ISSUE TITLE
Add gradient color? Animate color too?

### ISSUE THREAD

**Issue by MBach** [callstack/reanimated-arc#6]
Hi, 

Thank you for this library. I love react-native-paper by the way.

Is it possible to animate the color when animating the arc? If I follow the definition of "color", it seems that it's either a "string" or a "Reanimated.Node<number>". Can you provide an example please?

Is it possible to have a gradient color instead of just one color? The gradient should follow the curve of the arc of course.

**Comment by lucidprojects** [comment]
Sorry to open up an old question but @MBach did you ever find a solution for using a gradient color in the arc?

**Comment by MBach** [comment]
> Sorry to open up an old question but @MBach did you ever find a solution for using a gradient color in the arc?

Nope. Unfortunately, this repository is on hold, and not actively maintained anymore. But I know that people from callstack would review and merge any PR.

And right now, I don't have much time to do it by myself.
